### ISSUE TITLE
针对 webstorm 使用时组件属性并没有类型提示

### ISSUE THREAD

**Issue by yzl010318** [nutui-uniapp/nutui-uniapp#429]
### nutui-uniapp 版本号

1.7.17

### 平台

h5

### 重现链接

https://github.com/17774597536/uniapp/blob/main/src/pages/home/index.vue

### 重现步骤

拿按钮举例
![01](https://github.com/user-attachments/assets/3c1c5bca-239c-487a-b039-f7937e99507d)

### 期望的结果是什么？

显示类型提示 
![企业微信截图_20240926171318](https://github.com/user-attachments/assets/48af8e0f-ccc8-44fd-b6b6-f34a59db4004)

### 实际的结果是什么？

仅提示string
![02](https://github.com/user-attachments/assets/0ea7121d-2e76-4304-868a-478f81efef1f)

**Comment by xiaohe0601** [comment]
对 WebStorm 类型提示的支持是通过 `web-types` 实现的，目前文档中的表格没有独立拆分出 `可选值` 的列，需要重新整理一遍文档才行，这个工作量还挺大的😕

**Comment by yzl010318** [comment]
期待后续的更新
